mpd/playerstatus.go

```diff
@@ -47,4 +48,5 @@ func (p *PlayerStatus) Tick() {
 	diff := p.Since()
 	p.SetTime()
 	p.Elapsed += diff.Seconds()
+	p.ElapsedPercentage = float64(100) * p.Elapsed / float64(p.Time)
```

Comment:
I believe it possible that `p.Time` can be zero. A simple check would be inexpensive and future-proof.